base/static/base/css/base.css

```diff
@@ -12,15 +12,15 @@ strong {
 
 body {
   font-family: var(--primary-font);
-  background-color: rgb(255, 255, 255);
+  background-color: rgb(255 255 255);
   display: flex;
   flex-direction: column;
   gap: 20px;
-  padding: 20px 20px;
+  padding: 20px;
```

Comment:
![image](https://github.com/user-attachments/assets/20ce94d2-697a-4772-b700-df7560bc5444)
css has a bunch of little things that i just didn't know about, wow.